plugins/woocommerce/src/Admin/Features/LaunchYourStore.php

```diff
@@ -69,6 +69,11 @@ public function preload_settings( $settings ) {
 		$is_setting_page = $current_screen && 'woocommerce_page_wc-settings' === $current_screen->id;
 
 		if ( $is_setting_page ) {
+			// Regnerate the share key if it's not set.
+			if ( false === get_option( 'woocommerce_share_key', false ) ) {
```

Comment:
You could just use [add_option](https://developer.wordpress.org/reference/functions/add_option/) instead of wrapping an if statement. It has the same logic built in.

Reply:
Nice! I didn't know that. Thanks for the tip. Updated in 867a3678594bb6946dc44227c4e4940f1b052f9c